src/components/ChatLog.jsx

```diff
@@ -0,0 +1,35 @@
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatEntries = props.entries.map((entry) => {
+    return (
+      <ChatEntry
+        key={entry.id}
+        id={entry.id}
+        sender={entry.sender}
+        timeStamp={entry.timeStamp}
+        body={entry.body}
+        liked={entry.liked}
+        onLike={props.onLike}
+      />
+    );
+  });
+
+  return <main>{chatEntries}</main>;
```

Comment:
Nice wrapping this in the `<main>` tag. I would actually suggest to add in this `App.jsx` so you could wrap `ChatLog` in it as well. We don't show anything outside of `ChatEntry` components but if we did then those would be outside of our `<main>` tag. We could just keep moving them inside here but why do that when we could wrap the entire component.